README.md

```diff
@@ -1 +1,91 @@
-# glam_it_up
\ No newline at end of file
+![](https://img.shields.io/badge/GlamItUp-violet)
+
+# Space Traveler
+
+> Glam It Up website is built with real live data from the Make up API. The application displays a variety of beauty products such as lipsticks, bronzers, foundation among many other products. You can select any product type that is available and check it's description, price and name.
+
+![screenshot](./src/images/one.png)
+![screenshot](./src/images/two.png)
+
+[Live Link](https://ilynettes-glam-app.herokuapp.com/)
+
+[Loom Video](https://www.loom.com/share/738c4888330745798ab4f6e2895449f8)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages (HTML, CSS, JavaScript)
+
+- Frameworks / Libraries
+
+  ```bash
+  - React (Front end library)
+  - React Testing Library
+  - Jest(for testing)
+  - Git(version control)
+  - ESLint(JavaScript linting)
+  - Stylelint(style linting)
+  ```
+
+- Technologies used
+
+  ```bash
+  - Git(version control)
+  ```
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A text editor(preferably Visual Studio Code)
+
+### Install
+
+- [Git](https://git-scm.com/downloads)
+- [Node](https://nodejs.org/en/download/)
+
+### Usage
+
+#### Clone this repository
+
+```bash
+$ https://github.com/iLynette/glam_it_up.git
+$ cd glam_it_up
+```
+
+#### Run project
+
+```bash
+$ npm install
+$ npm start
+```
+
+## Author
+
+👤 **Lynette Aluoch Acholah**
+
+- GitHub: [@iLynette](https://github.com/iLynette)
+- Twitter: [@acholah_lynette](https://twitter.com/acholah_lynette)
+- LinkedIn: [lynette-acholah](https://linkedin.com/in/lynette-acholah)
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] In this `Acknowledgements` section kindly give an appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Thank you for the review, @elyor-doniyorov  i have resolved this